frontend/src/pages/AccountReceive.vue

```diff
@@ -256,6 +338,7 @@ function useScan() {
     startBlockLocal,
     userAddress,
     userAnnouncements,
+    tableKey,
```

Comment:
Can this be removed now?

Reply:
Yes, removed here: https://github.com/ScopeLift/umbra-protocol/pull/558/commits/599d33aead443ef144250df11c5d6f8714004b48